### PR TITLE
Add a verify target to dns-operator

### PR DESCRIPTION
**Pull request by danehans**
/assign @ironcladlou 
/cc @knobunc @Miciah @frobware

### CODE REVIEW

**Comment by ironcladlou** [comment]
Not sure about those errors — can you explain them?

File: ci-operator/jobs/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -355,3 +355,59 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )unit,?($|\s.*)
+  - agent: kubernetes
+      always_run: true
```
Comment (Miciah) [inline ci-operator/jobs/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master-presubmits.yaml:359]: The indentation here looks funny—`always_run` and its siblings should be at the same indentation level as `agent`.

**Comment by ironcladlou** [comment]
I don't know what went wrong with generation in this PR, but https://github.com/openshift/release/pull/8453 looks right

**Comment by ironcladlou** [comment]
Superceded by https://github.com/openshift/release/pull/8453

**Pull request closed without merging**